compute/cloud-client/src/main/java/compute/snapshotschedule/CreateSnapshotSchedule.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package compute.snapshotschedule;
+
+// [START compute_snapshot_schedule_create]
```

Comment:
Why is this sample duplicated?
It is already present in: https://github.com/GoogleCloudPlatform/java-docs-samples/pull/9742

Reply:
I need this classes for testing.

Reply:
Thank you! Deleted